src/corehost/cli/hostpolicy.cpp

```diff
@@ -54,6 +42,23 @@ int run(const arguments_t& args)
         return StatusCode::ResolverResolveFailure;
     }
 
+    pal::string_t clr_path = resolver.get_coreclr_path();
+    if (clr_path.empty() || !pal::realpath(&clr_path))
+    {
+        trace::error(_X("Could not resolve CoreCLR path. For more details, enable tracing by setting COREHOST_TRACE environment variable to 1"));;
+        return StatusCode::CoreClrResolveFailure;
+    }
+
+    pal::string_t clrjit_path = resolver.get_clrjit_path();
+    if (clrjit_path.empty() || !pal::realpath(&clrjit_path))
+    {
+        trace::warning(_X("Could not resolve CLRJit path, but CoreCLR might load the JIT if it is present side-by-side."));;
```

Comment:
I think this should be two different blocks:

if (jit path is empty) {warning: No explicit JIT path specified.}
else if (JIT path cannot be resolved via realpath) {warning: Possibly invalid JIT path.}

We should not call out CoreCLr policy (about JIT load) since it can change independent of the host.